bin/scpipe/lib/sc/pipe.rb

```diff
@@ -69,37 +71,32 @@ def prepare_pipe
       end
 
       def run_pipe
-        @@pipeproc = Proc.new {
-          trap("INT") do
-            Process.exit
-          end
-          rundir = Dir.pwd
+        rundir = Dir.pwd
+        while @done==false do
           IO.popen("cd #{rundir} && #{SC.sclang_path.chomp} -d #{rundir.chomp} -i scvim", "w") do |sclang|
-            loop {
```

Comment:
This is the loop based on the `cat` unix command that got replaced. Ruby 2.3 wasn't ending the `cat` processes after exiting pipe.rb, leading to a buildup of parentless `cat` processes. Stray cats!!! :)